mintpy/defaults/smallbaselineApp.cfg

```diff
@@ -183,6 +183,11 @@ mintpy.troposphericDelay.polyOrder      = auto  #[1 / 2 / 3], auto for 1
 mintpy.troposphericDelay.looks          = auto  #[1-inf], auto for 8, extra multilooking num
 mintpy.troposphericDelay.minCorrelation = auto  #[0.0-1.0], auto for 0
 
+## Notes for gacos:
+## GACOS products should be downloaded manually before running this step.
+## They have to be one .ztd binary file for each acquisition date,
+## and the path to these downloaded products should be given here:
+mintpy.troposphericDelay.gacosDir      = auto # [path2directory], auto for GACOS_DIR or "./"
```

Comment:
Since the GACOS products are named as (YY)YYMMDD.ztd format, it can not really be shared among different projects. I would recommend not use ${WEATHER_DIR}, but set it to `./GACOS` directly, inside the mintpy folder.

Could you please also add the GACOS website link and the relevant paper citation here so that user could easily refer to?